doc/source/data/doc_code/creating_datasets.py

```diff
@@ -0,0 +1,592 @@
+# flake8: noqa
```

Comment:
Could we enable flake8 to cap the line length?

Reply:
Maybe we should have some separate doc lint script checking line lengths overall.

Reply:
Can do, I was mindlessly cargo culting here. But can we also do this for all other example code? All other example code modules that I've seen disable flake8.

Reply:
Oh, I see why. We can't specify per file flake8 ignoring of certain rules (e.g. imports not being at the top of the module), and we don't want per-line flake8 rule ignoring since it would muddy the rendered example.

Reply:
@ericl Also, all of these lines are under 88 characters, which adheres to our flake8/black config, so I think we should update the code block rendering to be 88 characters wide to match our config.

Reply:
cc @maxpumperla

Reply:
@clarkzinzow for context, I think one of the primary reasons not to enforce the 88 char limit in docs overall is that special syntax like links (with long URLs) make this practically impossible.

Having said that, I think @ericl is right that we should do this for all the code we import, if possible. But there are some limitations, like the non-top-level imports that we simply can't get around. Especially for long examples it's pedagogically unsound to just throw huge list of imports at people at the beginning. Also, it's sometimes very useful to repeat an import statement from earlier, just to make a snippet copy-pastable.